vnext/src/Libraries/Components/View/ReactNativeViewViewConfig.windows.js

```diff
@@ -134,10 +134,13 @@ const ReactNativeViewConfig = {
   validAttributes: {
     ...ReactNativeViewViewConfigAndroid.validAttributes,
     accessibilityActions: true,
+    accessibilityDescribedBy: true,
```

Comment:
Windows custom attributes are at the bottom of this list with comments. Could you move these there?

Reply:
agree that this seems odd/wrong esp. if the uwp native code doesn't handle it

Reply:
Will restrict to win32